generator/typesystem_network.xml

```diff
@@ -171,11 +171,11 @@
   </value-type>
   <value-type name="QHttpHeaders" since-version="6.7"/>
   <enum-type name="QHttpHeaders::WellKnownHeader" since-version="6.7"/>
-  <!-- preliminary in 6.7
-  <value-type name="QNetworkRequestFactory" since-version="6.7"/>
-  <value-type name="QRestAccessManager" since-version="6.7"/>
-  <value-type name="QRestReply" since-version="6.7"/>
-  -->
+  <!-- preliminary in 6.7:  -->
```

Comment:
does the comment still make sense?

Reply:
I believe so, because these classes were available in 6.7, but currently only are generated for 6.8.